fuzzers/libafl-fuzz/src/feedback/persistent_record.rs

```diff
@@ -0,0 +1,161 @@
+use std::{
+    borrow::Cow,
+    collections::VecDeque,
+    fmt::{Debug, Formatter},
+    marker::PhantomData,
+};
+
+use libafl::{
+    corpus::{Corpus, Testcase},
+    events::EventFirer,
+    executors::ExitKind,
+    feedbacks::{Feedback, FeedbackFactory},
+    inputs::Input,
+    observers::ObserversTuple,
+    state::{HasCorpus, State},
+};
+use libafl_bolts::{Error, Named};
+use serde::{Deserialize, Serialize};
+
+/// A [`PersitentRecordFeedback`] tracks the last N inputs that the fuzzer has run.
```

Comment:
Just a thoguth, I don't think the last N always makes sense / you'd want all since restart (which, sadly, may be target dependent)

Reply:
@domenukk AFL++ does this based on a provided value. I think it's fine IMO.